lib/src/native/realm_core.dart

```diff
@@ -2647,6 +2652,14 @@ Pointer<realm_value_t> _toRealmValue(Object? value, Allocator allocator) {
   return realm_value;
 }
 
+Pointer<realm_binary_t> _toRealmBinary(List<int>? value, Allocator allocator) {
+  final realm_binary = allocator<realm_binary_t>();
+  if (value != null) {
+    realm_binary.ref.data = value.toUint8Ptr(allocator);
+    realm_binary.ref.size = value.length;
+  }
+  return realm_binary;
+}
```

Comment:
Is this used anywhere?

Reply:
It is not user anymore. I will delete this.